pkg/kusionctl/cmd/apply/options.go

```diff
@@ -314,38 +314,45 @@ func Apply(
 //	    return err
 //	}
 //
-//	Watch(o, kubernetesRuntime, planResources, changes)
+//	Watch(o, kubernetesRuntime, planResources, changes, os.Stdout)
 //	if err != nil {
 //	    return err
 //	}
 func Watch(o *ApplyOptions,
 	r runtime.Runtime,
 	planResources *models.Spec,
-	project *projectstack.Project,
-	stack *projectstack.Stack,
+	changes *opsmodels.Changes,
 	out io.Writer,
 ) error {
 	if o.DryRun {
 		fmt.Fprintln(out, "NOTE: Watch doesn't work in DryRun mode")
 		return nil
 	}
-	if !o.Watch {
```

Comment:
We don't need to check this watch option anymore?

Reply:
already checked in  line 152, so I remove the dup code